### PR TITLE
simpleitk: init at 1.1.0 (note this only builds the C++ libs, not the Python/Lua/... bindings)

### PR DESCRIPTION
**Pull request by bcdarwin**
I hope to add the Python and Lua bindings to the relevant package sets separately but depending on this package by checking out the SimpleITK source code but only making the relevant subdirectories.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [na Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [na] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [na] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/simpleitk/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, git, swig, lua, itk }:
+
+stdenv.mkDerivation rec {
+  _name    = "simpleitk";
```
Comment (FRidh) [inline pkgs/development/libraries/simpleitk/default.nix]: The use of the underscore is non-standard. Instead, use either a `let` expression or arguments that are commonly used, like `pname` and `version`.

Comment (bcdarwin) [inline pkgs/development/libraries/simpleitk/default.nix]: fixed.

**Comment by bcdarwin** [comment]
So I don't think there's any way to build just the bindings separately from the main build, which is annoying since I don't want to build separate versions of this package for all Python, Lua, etc., package sets.  Thoughts/relevant example packages in the Nix tree?

**Comment by Mic92** [comment]
Either multiple outputs or create multiple packages.

**Comment by matthewbauer** [comment]
@GrahamcOfBorg build simpleitk

**Review by matthewbauer — CHANGES_REQUESTED** [review]
See @GrahamcOfBorg log

**Comment by bcdarwin** [comment]
I'll try to look into this soon.

**Comment by bcdarwin** [comment]
@GrahamcOfBorg build simpleitk

**Comment by bcdarwin** [comment]
@matthewbauer updated version is building locally now.

**Pull request merged**